### DISCUSSION TITLE
Zigbee extender: Sonoff ZBMini?

### DISCUSSION THREAD

**Discussion by gforment** [indomus/forum#44] [ZigBee, Z-Wave, Thread, Matter]
Ciao,
stavo cercando un repeater per rete Zigbee da mettere all'esterno di casa dove ho una presa. Potrei mettere un Xiaomi Power Plug o qualsiasi connettore alimentato ma sporgerebbe dalla presa, quindi cercavo qualcosa di più discreto da mettere nella cassetta a muro: **Sonoff ZBMini** potrebbe andare bene da collegare a monte della presa? Lo terrei sempre acceso (presa sempre alimentata, non mi interessa controllarla). In alternativa funzionerebbe gli "IN" con 2 fili che partono dalla presa, senza mettere nessun carico in OUT? Suggerimenti o alternative?
Grazie!

**Comment by snakuzzo** [comment]
Il Sonoff ZBMini puoi tranquillamente farlo funzionare alimentandolo con fase e neutro e senza collegargli alcun carico.
Essendo un device alimentato in teoria dovrebbe funzionare anche da router, ma non ci metterei la mano sul fuoco. Posso fare una prova con l'unico che ho io...ti faccio sapere.

Ad ogni modo, anche in base alle esperienze di altri utenti, se non dovesse andar ben, il consiglio è quello di utilizzare come router un device con chip Texas Instruments (es. CC2530 o CC2531)

**Reply by gforment** [reply]
La menata è che con i TI devo installarlo su un device da tenere all'aperto, alimentare, gestire, ecc... preferivo qualcosa di più standalone

**Reply by snakuzzo** [reply]
Comunque il Sonoff ZBMini funziona anche da router

**Reply by ghost** [reply]
Io ho diversi Sonoff ZBMini con Zigbee2mqtt e funzionano correttamente anche come router.

**Reply by snakuzzo** [reply]
Giusto per info...qui di seguito i router sulla mia rete zigbee (tra cui puoi notare anche il sonoff zbmini)

![image](https://user-images.githubusercontent.com/7205371/112847194-1ddea200-90a7-11eb-9420-7740bc4b3931.png)

**Reply by gforment** [reply]
perfetto, il mio unico dubbio era messo in parallelo con una presa senza agganciare alcun carico sull'output.
Quindi procedo all'acquisto! 
Grazie ancora

**Reply by snakuzzo** [reply]
Considera che io ce l'ho da 2 giorni collegato con una spina "volante" :D
